packages/components/src/utils/button-handling.spec.ts

```diff
@@ -0,0 +1,80 @@
+import * as handleButtonUtils from './button-handling';
+import { handleButtonEvent, improveButtonHandlingForCustomElement } from './button-handling';
+
+describe('improveButtonHandlingForCustomElement()', () => {
+  test('it should add a click event listener to the element', () => {
```

Comment:
I think we always use the `it` alias instead of `test` and then the description starts with `should..`.